helm/adelphi/templates/gemini-job.yaml

```diff
@@ -57,8 +59,9 @@ spec:
       args:
         - echo "Extract Gemini schema using anonymizer script";
           pip install cassandra-driver;
-          wget -q https://raw.githubusercontent.com/datastax/adelphi/master/schema-anonymizer/anonymizer.py;
-          python anonymizer.py --hosts="{{ .Values.source.clusterName }}-{{ .Values.source.dc }}-service" --username="{{ .Values.source.clusterName }}-superuser" --password="`cat /source-secret/password`" --keyspaces="{{ .Values.gemini_schema_keyspace }}" --format=gemini --no-anonymize > /workspace/gemini_schema.json;
+          cd /workspace/adelphi/python/adelphi;
+          python3 setup.py install;
```

Comment:
Semi-nit: probably prefer "pip install ." here (along the same lines as similar discussions on other PRs).

Reply:
Fixed